infra/managed-cluster/aks-cluster.bicep

```diff
@@ -284,6 +284,16 @@ module deployAKS 'br/SharedDefraRegistry:container-service.managed-clusters:0.5.
               validation: 'none'
               prune: true
             }
+            services: {
+              path: fluxConfig.clusterCore.kustomizations.servicesPath
```

Comment:
Aimee, please add the servicesPath in parameters.json and services kustomization should depend on both cluster and infra.

Reply:
You will need to also modify the parameters file. In the fluxConfig, there is an extra configuration "services" that is no longer required. The path for services is still required 

"servicesPath": "./services/#{{ lower(environment) }}/01"